### DISCUSSION TITLE
Directors & Actors listing page

### DISCUSSION THREAD

**Discussion by valerebron** [jellyfin/jellyfin#6708] [General]
**Directors & Actors** should be **emphasis**, people pages could be a good suggestion.
I propose 2 links in the tabs section of a movie library.
These links should redirect to a page where directors are listed like in the cast & crew section.

What do you think about that ?

**Comment by NiklasBr** [comment]
I've been looking for something like this. Would be awesome if there was a way to do this!

**Reply by valerebron** [reply]
It's easy toimplement a basic version with node, but it should be better to get a page inside the app.
I can share you my code if you like ...

**Reply by NiklasBr** [reply]
I would like to see it if you have the time to do it. Might even be worth creating a plugin with that as a basis?

**Reply by valerebron** [reply]
you can check this repo (really basic but it works) :
https://github.com/valerebron/jellyfin-directors
